README.md

```diff
@@ -880,7 +880,6 @@ These plugins add tab completions without adding extra functions or aliases.
 * [startify](https://github.com/zdharma/zsh-startify) - vim-startify -like plugin for ZSH.
 * [surf](https://github.com/beardcoder/surf.plugin.zsh) - Add completions for surf.
 * [symphony (TheGrowingPlant)](https://github.com/TheGrowingPlant/symfony.plugin.zsh) - Autocompletion for Symfony 3 and 4 commands.
-* [test-kitchen-zsh-plugin](https://github.com/pelletiermaxime/test-kitchen-zsh-plugin) - Add completions for [Test Kitchen](https://kitchen.ci/).
```

Comment:
I know they're in the process of getting the domain situation cleaned up, but it's been weeks and I'm tired of the test fails too. Instead of yanking it entirely, please delink kitchen.ci and replace the it with **Test Kitchen**.

I can relink it once kitchen.ci is sorted out.

Thanks for the contribution, every bit helps

Reply:
The plugin still exists and is still good, and the kitchen software still exists, it's just the website that's busted because of the domain issue.

Reply:
I changed the link to point to their github project instead.